src/en/charms-constraints.md

```diff
@@ -235,28 +254,19 @@ spaces, and not connected to either the 'storage' or 'dmz' spaces.
 
 See the [Network spaces][network-spaces] page for details on spaces.
 
-To get exactly two CPUs for a machine in a localhost cloud:
+To get a maximum of two CPUs for a machine in a LXD cloud:
```

Comment:
To create a machine with two CPUs for a machine in a LXD cloud:

Reply:
How about

"For a LXD cloud, to create a machine with two CPUs:"

Repeating "a machine" is awkward.

Reply:
fair enough on the double machine. I think we can let go of the "maximum" and calling out lxd apart somewhere through here. It's worth a note but I think it'd be worth letting go of it and treating them as common vocabulary through the doc.